simapp/app.go

```diff
@@ -233,12 +233,12 @@ func NewSimApp(
 
 	// Create IBC Keeper
 	app.IBCKeeper = ibc.NewKeeper(
-		app.cdc, keys[ibc.StoreKey], app.StakingKeeper, scopedIBCKeeper,
+		app.cdc, ibc.NewAnyCodec(appCodec), keys[ibc.StoreKey], app.StakingKeeper, scopedIBCKeeper,
```

Comment:
@fedekunze why do you need `app.cdc` here? What methods do you need to use on it? JSON?

Reply:
Nvm, it's for Tendermint types. Let's add a comment stating that this should be removed once new Tendermint version is released.